dora/core/common/src/main/java/alluxio/MembershipType.java

```diff
@@ -0,0 +1,10 @@
+package alluxio;
+
+/**
+ * MembershipManager type
```

Comment:
what does each different type mean? e.g., I am not sure I understand what NONE means here.

Reply:
STATIC requires a file path provided, ETCD requires an ETCD, NONE is for testing purposes. Or do you suggest i make STATIC as default and check in a sample static file sits within alluxio home dir?

Reply:
I  was suggesting simple javadoc here to explain the difference across these types.  e.g.

```java
  STATIC, // a static file of all workers registration
  ETCD,  // worker register on ETCD
  NONE // Noop (for debugging purpose???)
```

Reply:
done